AniPendasheva/src/main/java/task2/Envelope.java

```diff
@@ -0,0 +1,65 @@
+package task2;
+
+import java.util.Scanner;
+
+import static task2.Main.inputValues;
+
+
+public class Envelope {
+    private float height;
+    private float width;
+
+    public Envelope() {
+        this.height = 0;
+        this.width = 0;
+    }
+
+    boolean isFittable(Envelope envelope) {
+        if (this.height >= envelope.height && this.width >= envelope.width) {
+            return true;
+        }
+        if (this.height >= envelope.width && this.width >= envelope.height) {
+            return true;
+        }
+        if (this.height <= envelope.height && this.width <= envelope.width) {
+            return true;
+        }
+        if (this.height <= envelope.width && this.width <= envelope.height) {
+            return true;
+        }
+
```

Comment:
I have a doubt about the correct operation of this code. Please test on different values. As an example (5x9) and (2x6), (9x5) and (6x2).